### PR TITLE
 update readme.md and add `--metricsaddr` cmd args 

### PR DESCRIPTION
**Pull request by j4ckzh0u**
update readme.md and add `--metricsaddr` cmd args

### CODE REVIEW

**Comment by cuisongliu** [comment]
image
this commit is behind of main code...

**Comment by j4ckzh0u** [comment]
squash some commits

**Review by cuisongliu — COMMENTED** [review]
change review

File: controllers/run_probe.go [hunk 1]
```diff
@@ -67,11 +69,12 @@ func (w *work) doProbe() (keepGoing bool) {
 
 	// the full container environment here, OR we must make a call to the CRI in order to get those environment
 	// values from the running container.
-	result, output, err := proberCheck.runProbeWithRetries(w.p, w.retry)
+	result, output, err, checkProbe := proberCheck.runProbeWithRetries(w.p, w.retry)
```
[the remark this answers is not in the document]
Comment (j4ckzh0u) [reply controllers/run_probe.go:72]: ok, thx, I will fix it!

File: cmd/app/options/options.go [hunk 2]
```diff
@@ -32,6 +32,7 @@ type Options struct {
 	LeaderElectionResourceLock string
 	MaxConcurrent              int
 	MaxRetry                   int
+	MetricsBindAddress         string
```
[the remark this answers is not in the document]
Comment (j4ckzh0u) [reply cmd/app/options/options.go:35]: this item value is ":9090" or "0.0.0.0:9090", I think address is better.

[the remark this answers is not in the document]
Comment (j4ckzh0u) [reply cmd/app/options/options.go]: ok, I have already fix it.

File: controllers/sync.go [hunk 3]
```diff
@@ -142,11 +145,18 @@ func (c *Reconciler) syncEndpoint(ctx context.Context, cep *v1beta1.ClusterEndpo
 	}
 }
 
-func healthyCheck(host string, cep *v1beta1.ClusterEndpoint, retry int) ([]v1beta1.ServicePort, []error) {
+func healthyCheck(host string, cep *v1beta1.ClusterEndpoint, retry int, metricsinfo *metrics.MetricsInfo) ([]v1beta1.ServicePort, []error) {
 	var wg sync.WaitGroup
 	var mx sync.Mutex
 	var data []v1beta1.ServicePort
 	var errors []error
+	checkListChan := make(chan struct {
```
[the remark this answers is not in the document]
Comment (j4ckzh0u) [reply controllers/sync.go]: tihis commit is use []struct.

**Pull request merged**